lc-hax/Scripts/Commands/SellCommand.cs

```diff
@@ -16,12 +19,68 @@ public class SellCommand : ICommand {
             return;
         }
 
-        HaxObjects.Instance?.GrabbableObjects.ForEach(nullableGrabbableObject => {
-            if (!nullableGrabbableObject.IsNotNull(out GrabbableObject grabbableObject)) return;
-            if (!grabbableObject.itemProperties.isScrap) return;
+        if (args.Length is 0) {
+            SellEverything();
+            return;
+        }
+
+        if (args.Length is 1 && int.TryParse(args[0], out int targetAmount)) {
+            SellScrapAmount(targetAmount);
+            return;
+        }
+
+        void SellEverything() {
+            HaxObjects.Instance?.GrabbableObjects.ForEach(nullableGrabbableObject => {
+                if (!nullableGrabbableObject.IsNotNull(out GrabbableObject grabbableObject)) return;
+                if (!grabbableObject.itemProperties.isScrap) return;
+                if (grabbableObject.itemProperties.isDefensiveWeapon) return;
```

Comment:
Are some defensive items assigned as scrap?

Reply:
Yeah found out the shotgun is marked as scrap as well